content/en/database_monitoring/troubleshooting.md

```diff
@@ -279,7 +279,7 @@ Some or all queries may not have plans available. This can be due to unsupported
 | The Agent is running an unsupported version. | Ensure that the Agent is running version 7.30.0 or greater. We recommend regular updates of the Agent to take advantage of new features, performance improvements, and security updates. |
 | The Agent is not able to execute a required function in the `datadog` schema of the database. | The Agent requires the function `datadog.explain_statement(...)` to exist in **all databases** the Agent can collect queries from. Ensure this function was created by the root user according to the [setup instructions][1] and that the `datadog` user has permission to execute it. |
 | Queries are truncated. | See the section on [truncated query samples](#query-samples-are-truncated) for instructions on how to increase the size of sample query text. |
-| The application client used to execute the query is using the Postgres extended query protocol or prepared statements. | Some client applications using the Postgres [extended query protocol][2] do not support the collection of explain plans due to the separation of the parsed query and raw bind parameters. For instance, clients such as the Go client [sqlx][3] and Python client [asyncpg][4] use the extended query protocol. To work around this limitation, you can modify your application to send the raw SQL queries including bind parameters. |
+| The application client used to execute the query is using the Postgres extended query protocol or prepared statements. | Some client applications using the Postgres [extended query protocol][2] do not support the collection of explain plans due to the separation of the parsed query and raw bind parameters. For instance, clients such as the Go client [sqlx][3] and Python client [asyncpg][4] use the extended query protocol. To work around this limitation, you can update your database client to use the simple query protocol. For example, setting `preferQueryMode = simple` for the [Postgres JDBC Client](https://jdbc.postgresql.org/documentation/head/connect.html) will ensure queries are sent using the simple query protocol. |
```

Comment:
@djova Thanks for the edits! Just a heads up that this link can't be inlined as we translate our documentation and that process requires a numbered list of links. Suggested edit would be to create a new link at the bottom of the list.

Reply:
Oh forgot about the link inlining. Updated!